{"name": "Проверка: если не передать query-параметр с треком, вернётся ошибка 400", "status": "passed", "start": 1710697567250, "stop": 1710697567754, "uuid": "02c4e029-1811-4f98-8284-db32039c839d", "historyId": "dcf6439dd6b010d7594a42cd2c9df220", "testCaseId": "dcf6439dd6b010d7594a42cd2c9df220", "fullName": "tests.tests_order.test_get_order_by_track.TestGetOrderByTrack#test_get_order_without_track_causes_400_error", "labels": [{"name": "parentSuite", "value": "tests.tests_order"}, {"name": "suite", "value": "test_get_order_by_track"}, {"name": "subSuite", "value": "TestGetOrderByTrack"}, {"name": "host", "value": "Olyas-MacBook-Air.local"}, {"name": "thread", "value": "94503-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_order.test_get_order_by_track"}]}